en/organisation/sammansattning/body.md

```diff
@@ -1,179 +1,100 @@
 
-__Contact__
+**Contact**
 
 The Secretary can be reached at [sekreterare@datasektionen.se](mailto:sekreterare@datasektionen.se).
 
-__Contact person to__
-
-* Nominations committee chairperson
-
-* Nominations committee members
+## Board member of Business Relations - Adam Sjöberg
 
-* Student union assembly delegates
-
-## Board Member of Educational Affairs - Abhinav Sasikumar
-
-This board member works closely with the study council and works as a
-link between them and the board.
-
-__Contact__
-
-The Board Member of Educational Affairs can be reached at [d-uf@datasektionen.se](mailto:d-uf@datasektionen.se).
-
-__Contact person to__
-
-* Program responsible student
-
-* The Chairman of the Study Council
-
-* The Chairman of the Equality Committee
-
-* International Student Coordinator
+Works with business related question and our projects and committees related to this area.
+Is also responsible for internal communications and that important information is available to our members.
 
-* Student Safety Officer
+**Contact**
 
-* Project Leader for Project Pride
+The Board member of Business Relations can be reached at [d-nok@datasektionen.se](mailto:d-nok@datasektionen.se).
 
 ## Board member of Student Issues - Malin Svenberg
 
 Works with question regarding the well being of our members, working environment, and
 social events.
 
-__Contact__
+**Contact**
 
 The Board member of Student Issues can be reached at [d-ssf@datasektionen.se](mailto:d-ssf@datasektionen.se).
 
-__Contact person to__
-
-* Head of the Bar
-
-* Chapter Sports Leader
+## Board member of Educational Affairs - Abhinav Sasikumar
 
-* Light and Sound responsible 
-
-* Chief of Halls
-
-* Cultural Attaché
-
-* Prylis
-
-* Mulle/Mullerina Schmeck
-
-* DESCtop
-
-* Ärkedemon
-
-* Chair of Ada
-
-* Spexdireqteur
-
-* Project leader of dÅre
-
-* Bakis
-
-* D-Fest
-
-* Kingpin
-
-* Project Leader for Hello World
-
-* Father/Mother Christmas
-
-* Choir Master
-
-* Song Responsible Person
-
-## Board Member of Business Relations - Adam Sjöberg
-
-Works with business related question and our projects and committees related to this area.
-Is also responsible for internal communications and that important information is available to our members.
-
-__Contact__
-
-The Board member of Business Relations can be reached at [d-nok@d.kth.se](mailto:d-nok@d.kth.se).
+This board member works closely with the study council and works as a
+link between them and the board.
 
-__Contact person to__
+**Contact**
 
-* Project Leaders for Studs
+The Board member of Educational Affairs can be reached at [d-uf@datasektionen.se](mailto:d-uf@datasektionen.se).
 
-* Chapter Systems Officer
 
-* Communications Officer
+## Board member of Study environment and Equality
 
-* Head of Business Relations
+Has overall responsibility for all issues connected to the physical and psychological study environment. Is the board's contact for questions relating to these areas. These responsibilities include
```

Comment:
The translation of the beginning of this part feels a bit off.
```suggestion
Has an overall responsibility for all issues connected to the physical and psychological study environment. Is the person to contact on the board for questions relating to these areas. These responsibilities include
```

Reply:
You're right, in my defense it was very late when I wrote it :sweat_smile: